android/src/main/java/com/tailscale/ipn/QuickToggleService.java

```diff
@@ -9,6 +9,7 @@
 import android.service.quicksettings.Tile;
 import android.service.quicksettings.TileService;
 
+@SuppressWarnings("deprecation")
```

Comment:
can you move this to the line with the deprecated API and if relevant, add a migration todo?